dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_28__Make_ou_event_and_enrollment_not_null.sql

```diff
@@ -0,0 +1,37 @@
+DO $$
+DECLARE dummyOrgUnitId bigint;
+DECLARE dummyOrgUnitUid varchar(11);
+BEGIN
+    select coalesce((select max(organisationunitid) + 1 from organisationunit), 1) into dummyOrgUnitId;
+    select generate_uid() into dummyOrgUnitUid;
+
+    while (select count(*) from organisationunit where uid = dummyOrgUnitUid) > 0 loop
+        select generate_uid() into dummyOrgUnitUid;
+    end loop;
+
+    insert into organisationunit
+    (organisationunitid, name, code, parentid, shortname, openingdate, created, lastupdated, uid, hierarchylevel)
```

Comment:
I am thinking of several things here.
1. Is there some unique constraint on the name or code? Maybe not.
2. I am thinking if hierarchylevel=99 will have any consequences? Since it will also not have a parent id, it will simply be an orphan. 
3. Functionally we might be reliant on the path property, in this case path is null which we might have not handled?